server/models/user.coffee

```diff
@@ -0,0 +1,14 @@
+americano = require 'cozydb'
+
+
+module.exports = User = americano.getModel 'User',
+    onboardedSteps: Array
+
+
+User.first = (callback) ->
+    User.request 'all', (err, users) ->
+        if err then callback err
+        else if not users or users.length is 0 then callback null, null
+        else
+            user = users[0]
```

Comment:
There is a bug with cozy-db, the `onboardedSteps` property returned by cozy-db is an array of array instead of an array.

We handled it in cozy-proxy here https://github.com/cozy/cozy-proxy/blob/development/server/models/user.coffee#L83 and here https://github.com/cozy/cozy-proxy/blob/development/server/models/user.coffee#L28